terraform/modules/common/ami/main.tf

```diff
@@ -0,0 +1,25 @@
+data "aws_ami" "source" {
+  most_recent = true
+
+  filter {
+    name   = "name"
+    values = ["ubuntu/images/hvm/ubuntu-${var.ubuntu_release}-*"]
```

Comment:
our existing images are amazon linux - can we stick to that? (we went with it to have the baked-in optimized drivers for networking and i/o)